lib/Parser/Parse.cpp

```diff
@@ -12371,9 +12245,9 @@ ParseNode* Parser::CopyPnode(ParseNode *pnode) {
       // General nodes.
       //PTNODE(knopVarDecl    , "varDcl"    ,None    ,Var  ,fnopNone)
     case knopVarDecl: {
-      ParseNode* copyNode=CreateNodeT<knopVarDecl>(pnode->ichMin,pnode->ichLim);
-      copyNode->AsParseNodeVar()->pnodeInit=CopyPnode(pnode->AsParseNodeVar()->pnodeInit);
-      copyNode->AsParseNodeVar()->sym=pnode->AsParseNodeVar()->sym;
+      ParseNodeVar* copyNode= Anew(&m_nodeAllocator, ParseNodeVar, knopVarDecl, pnode->ichMin, pnode->ichLim, nullptr);
+      copyNode->pnodeInit=CopyPnode(pnode->AsParseNodeVar()->pnodeInit);
```

Comment:
Nitpick on the spacing in these couple lines - if you clean up some of this should clean up all of it.

Reply:
Yeah... Just don't want to introduce a whole bunch of space diffs.  Next change.